ui/src/App.vue

```diff
@@ -49,7 +49,8 @@ export default class App extends Vue {
   @appNS.State('messages') messages!: Message[]
 
   get release (): string {
-    return process.env.VUE_APP_RELEASE || '?'
+    const commit = process.env.VUE_APP_COMMIT?.slice(0, 7) ?? 'dev'
```

Comment:
8.. okay at least they are not random (-:

Reply:
7 characters is quite standard when referencing git commits. And no, not random ;)